examples/torch/common/model_loader.py

```diff
@@ -26,6 +30,9 @@
 from nncf.torch.utils import safe_thread_call
 
 
+MODELS_PATH = '~/.cache/nncf/models'
```

Comment:
Bring this in line with the general approach in https://github.com/openvinotoolkit/nncf/pull/1579

Reply:
Seems like I need #1579 to be merged first. Maybe better to merge this PR and change `MODELS_PATH` in #1579 ?